packages/DataTable/src/helpers/rowHeightHelper.ts

```diff
@@ -0,0 +1,78 @@
+import { TableDataItemType, TableColumnsWidth } from "../types";
+
+const DEFAULT_HEIGHT = 40;
+// TODO: match desgin
+const DEFAULT_STYLES = "padding: 5px;border-right:1px solid;border-bottom:1px solid;";
+
+const containerId = `paprika-data-table__test-area--${Date.now()}`;
+
+interface ConstructorOptions {
+  defaultHeight?: number;
+  styles?: string;
+}
+
+export default class RowHeightHelper {
```

Comment:
Was there a need for this to be a class?

Reply:
Because users can potentially customize the default values, so `new Helper(options)` may make more sense